packages/aztec.js/src/note/index.js

```diff
@@ -307,10 +307,12 @@ export function encodeMetadata(noteArray) {
  * @returns {Note} created note instance
  */
 export async function fromEventLog(logNoteData, spendingKey = null) {
-    const publicKey = noteCoder.decodeNoteFromEventLog(logNoteData);
-    const newNote = new Note(publicKey, null);
+    const notePublicKey = noteCoder.decodeNoteFromEventLog(logNoteData);
+    const newNote = new Note(notePublicKey, null);
     if (spendingKey) {
         await newNote.derive(spendingKey);
+        const { publicKey } = secp256k1.accountFromPrivateKey(spendingKey);
+        newNote.owner = publicKey;
```

Comment:
I think the note owner is the Ethereum address of the owning account rather than `publicKey`? On the `note` class:
```
        /**
         * Ethereum address of note's owner
         * @member {string}
         */
        this.owner = owner;
```

Reply:
Ah, good catch!